chemprop/cli/hpopt.py

```diff
@@ -115,15 +115,10 @@ def add_hpopt_args(parser: ArgumentParser) -> ArgumentParser:
         type=str,
         nargs="+",
         default=["basic"],
-        help=f"""The model parameters over which to search for an optimal hyperparameter configuration.
-    Some options are bundles of parameters or otherwise special parameter operations.
-
-    Special keywords:
-        basic - the default set of hyperparameters for search: depth, ffn_num_layers, dropout, message_hidden_dim, and ffn_hidden_dim.
-        learning_rate - search for max_lr, init_lr_ratio, final_lr_ratio, and warmup_epochs. The search for init_lr and final_lr values
-            are defined as fractions of the max_lr value. The search for warmup_epochs is as a fraction of the total epochs used.
-        all - include search for all 13 inidividual keyword options
-
+        help=f"""The model parameters over which to search for an optimal hyperparameter configuration. Some options are bundles of parameters or otherwise special parameter operations. Special keywords include:
+        - ``basic``: Default set of hyperparameters for search (depth, ffn_num_layers, dropout, message_hidden_dim, and ffn_hidden_dim)
+        - ``learning_rate``: Search for max_lr, init_lr_ratio, final_lr_ratio, and warmup_epochs. The search for init_lr and final_lr values are defined as fractions of the max_lr value. The search for warmup_epochs is as a fraction of the total epochs used.
+        - ``all``: Include search for all 13 individual keyword options (including: activation, aggregation, aggregation_norm, and batch_size which aren't included in the other two keywords).
     Individual supported parameters:
         {list(DEFAULT_SEARCH_SPACE.keys())}
```

Comment:
There's an out-standing bug here. If `ray` isn't installed, the keys of `DEFAULT_SEARCH_SPACE` are never populated. Since it's an optional install not required when the docs are built, the "Individual supported parameters" is always empty.

We could just leave this here until `ray` becomes a mandatory install, or ignore it and try to fix it later, or just comment it out for now (it would be nice to have a persistently updated list of search space keys in case it changes later)

Reply:
Actually this issue looks like it'll be addressed by #918, so no need to worry about it here.